application/common/config/main.php

```diff
@@ -48,10 +48,11 @@
 $idBrokerConfig['validIpRanges'] = Env::getArray('ID_BROKER_validIpRanges');
 
 $zxcvbnApiBaseUrl = Env::get('ZXCVBN_API_BASEURL');
-$passwordRules = Env::getArrayFromPrefix('PASSWORD_RULE_');
-$passwordRules['minLength'] = $passwordRules['minLength'] ?? null;
-$passwordRules['maxLength'] = $passwordRules['maxLength'] ?? null;
-$passwordRules['minScore'] = $passwordRules['minScore'] ?? null;
+
+$passwordRulesEnv = Env::getArrayFromPrefix('PASSWORD_RULE_');
+$passwordRules['minLength'] = $passwordRulesEnv['minLength'] ?? 10;
```

Comment:
It looks like `$passwordRules` is never initialized. That still works in PHP, but it might be worth changing...
```php
$passwordRules['minLength'] = $passwordRulesEnv['minLength'] ?? 10;
$passwordRules['maxLength'] = $passwordRulesEnv['maxLength'] ?? 255;
$passwordRules['minScore'] = $passwordRulesEnv['minScore'] ?? 3;
```
... to something like...
```php
$passwordRules = [
    'minLength' => $passwordRulesEnv['minLength'] ?? 10,
    'maxLength' => $passwordRulesEnv['maxLength'] ?? 255,
    'minScore' => $passwordRulesEnv['minScore'] ?? 3,
];
```
Just a suggestion.

Reply:
Thanks, I'll make this edit.